test/unit/lookup_value_test.rb

```diff
@@ -154,13 +154,31 @@ def valid_attrs2
     assert value.valid?
   end
 
-  test "boolean lookup value should not allow for nil" do
+  test "boolean lookup value should not allow for nil value" do
     #boolean key
     key = lookup_keys(:three)
     value = LookupValue.new(:value => nil, :match => "hostgroup=Common", :lookup_key_id => key.id)
     refute value.valid?
   end
 
+  test "lookup value should allow valid key" do
```

Comment:
Could you also add a test please for multi-value lookup values?  (hostgroup=Common,domain=foo)